lib/cyclonedx/report_ruby_gems_cyclonedx.rb

```diff
@@ -1,21 +1,18 @@
 module Cyclonedx
   class ReportRubyGems < Base
+    TYPE = "gem".freeze
+
     def initialize(scan_report)
       super(scan_report)
     end
 
-    def package_url(dependency)
-      "pkg:#{dependency[:type]}/#{dependency[:name]}#{version_string(dependency, true)}"
-    end
-
     # Return version string to be used in purl or component
     def version_string(dependency, is_purl_version = false)
```

Comment:
This looks like the same logic as the def version_string above... you may wanna DRY this up

Reply:
Agreed, made a change to refactor the `version_string` method to the super class